Test/ExternalSubscriptionTest.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Xunit;
+
+namespace Recurly.Test
+{
+    public class ExternalSubscriptionTest : BaseTest
+    {
+        [RecurlyFact(TestEnvironment.Type.Integration)]
+        public void LookupExternalSubscription()
+        {
+            var uuid = "rv1loyqlf6si";
+            var externalSubscription = ExternalSubscriptions.Get(uuid);
+            externalSubscription.Quantity.Should().Be(14);
```

Comment:
Do you know why getting a thing by uuid is returning a collection of 14 elements? is the external subscription here just a composition of it's methods?

Reply:
The `Quantity` is a field on ExternalSubscription. Not the quantity of external subscriptions received. 14 is the quantity I chose when I made this ExternalSubscription